File: Python Programming/problems on string manipulation/problem100.py
'''
98. Write a program which accept string from user and return difference between frequency of small characters and frequency of capital characters. 
	
	Input :   "MarvellouS"
	Output :   6 (8-2)  
		
	Author : Milind Nalawade
	
	Date : 13/07/2021
	
	Language : Python 3.9.0

'''
#====
#Logic
#====
def Reverse(str):
	no1=no2=0
	for i in reversed(str):
		print(i,end="")
				
#=============
#Entry point function
#=============
def main():
	str=input("\n Enter your string : ")
	Reverse(str)
#=====
#starter
#=====
if __name__=="__main__":
	main()

File: C Programming/problems on strings maipulation/problem100.c
/*
100. Write a program which accept string from user and display it inn reverse order. 
	
	Input :   “MarvellouS” 
	Output :   “SuollevraM”
	Author : Milind Nalawade
	
	Date : 13/07/2021
	
	Language : C programming
	
*/
#include<stdio.h>
#include<string.h>
//Logic
void Reverse(char *str)
{
	 int size=strlen(str);
	 printf(" ");
	for(int i=size-1;i>=0;i--)
	{
		printf("%c",str[i]);
	}
}

//Entry point function

int main()	
{
	char arr[20];
	int ret=0;
	printf("\n Enter your string : ");
	gets(arr);
	Reverse(arr);
	return 0;
}	